applications/credhub-api/src/main/resources/db/migration/mysql/V57_2__delete_related_encrypted_value_when_credential_is_deleted.sql

```diff
@@ -0,0 +1,50 @@
+-- Fix for https://github.com/cloudfoundry/credhub/issues/231
+-- Create delete trigger on credential table instead of the tables directly
```

Comment:
It's not quite clear when I first read this which tables "instead of on the tables directly" refers to.

Reply:
I think that the SQL code itself shows what tables we are talking about here. Do you think that we should list table names in the comment? BTW, it has more comments that most of the existing flyway migration scripts. We usually do not have any comments.

Reply:
I will rephrase my argument. The comment combined with the code itself is more than sufficient.